rllib/evaluation/rollout_worker.py

```diff
@@ -420,7 +420,7 @@ def make_env(vector_index):
         if (ray.is_initialized()
                 and ray.worker._mode() != ray.worker.LOCAL_MODE):
             # Check available number of GPUs
-            if not ray.get_gpu_ids(as_str=True):
+            if not ray.get_gpu_ids():
```

Comment:
Please revert this.

Reply:
I'm getting warnings that by default, it'll be `as_str` anyways. Just tried to fix these.

Reply:
Resolved. `as_str` is deprecated.